### PR TITLE
added fact

### PR DESCRIPTION
**Pull request by rachel-emsley**
## Purpose
We need more facts!

## Approach
Added a new fact to the list, please review.

## Checklist
_Reviewer to complete_
- [x] Code conforms to coding standards
- [x] Changes made resolve the problem or meet the acceptance criteria of the feature
- [x] Able to run the code locally
- [x] Existing and new tests pass
- [x] Documentation has been updated

### CODE REVIEW

**Review by rachel-emsley — COMMENTED** [review]
Nice work Dave

File: facts.md [hunk 1]
```diff
@@ -4,3 +4,4 @@
 - A group of cats is called a clowder of cats
 - The evolution and use of JavaScript, a language developed in 10 days back in 1995!
 - Dreamt, along with it's variations, is the only word in the english language that ends in "mt".
+- There are Nine million bicycles in Beijing
```
Comment (rachel-emsley) [inline facts.md]:
```suggestion
- There are nine million bicycles in Beijing
```

File: facts.md [hunk 2]
```diff
@@ -4,3 +4,4 @@
 - A group of cats is called a clowder of cats
 - The evolution and use of JavaScript, a language developed in 10 days back in 1995!
 - Dreamt, along with it's variations, is the only word in the english language that ends in "mt".
+- There are nine million bicycles in Beijing
```
Comment (rachel-emsley) [inline facts.md:7]:
```suggestion
- There are nine million bicycles in Beijing.
```

**Review by Cloud-Monkey — APPROVED** [review]

**Review by rachel-emsley — COMMENTED** [review]
👍

**Pull request merged**